test/e2e/tests/account/snap-account-signatures.spec.ts

```diff
@@ -62,8 +59,22 @@ describe('Snap Account Signatures @no-mmi', function (this: Suite) {
           await experimentalSettings.check_pageIsLoaded();
           await experimentalSettings.toggleRedesignedSignature();
 
-          // Run all 5 signature types
+          // Connect the SSK account
           await new TestDapp(driver).openTestDappPage();
+
+          await driver.findClickableElement({ text: 'Connect', tag: 'button' });
```

Comment:
Nit: I see that the changes here are different than the prior file, despite the changes being similar in nature. The previous file uses `connectAccount` on the test dapp instance rather than running these connect steps one by one. Worth considering which approach we prefer, if they are indeed equivalent.

Reply:
resolved in this PR: https://github.com/MetaMask/metamask-extension/pull/29761/files#diff-9b81efe5f691fe688a21758fc8d6bd7a8dbdb58f0918b9da50e49984dd46743d

Reply:
i missed this, sorry. addressed in that PR here https://github.com/MetaMask/metamask-extension/pull/29761/commits/fc842e9bfe3729e6d0e8147ab0563b66972582f1